### PR TITLE
Added boilerplate for "my community stakes" page 

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/7050

## Description of Changes
- Added boilerplate code for "my community stakes" page
- Added `FLAG_MY_COMMUNITY_STAKE_PAGE_ENABLED` to hide the "my community stakes" page components

## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_MY_COMMUNITY_STAKE_PAGE_ENABLED` flag in `.env`
- Visit `/myCommunityStake`
- Verify you see this UI, and verify this is responsive across different screen sizes
image

## Deployment Plan
N/A

## Other Considerations
N/A

### CODE REVIEW

**Comment by lzach83** [comment]
Works locally, looking at code now

File: packages/commonwealth/client/scripts/views/pages/MyCommunityStake/MyCommunityStake.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+              label: filter,
+            }))}
+            value={{ label: activeFilter, value: activeFilter }}
+            onChange={(option) => setActiveFilter(option.value)}
+          />
+        </div>
+      </section>
+
+      <CWTabsRow>
+        {Object.values(TABS).map((tab, index) => (
```
Comment (lzach83) [inline packages/commonwealth/client/scripts/views/pages/MyCommunityStake/MyCommunityStake.tsx:57]: I think this causes some redundancy, `Object.values` creates an array then .map generates a new array of jsx. 

The data set is static in an Object, so i think we have room to just make the data set an array and eliminate the extra conversion.

**Review by lzach83 — CHANGES_REQUESTED** [review]
Small change then gtg

File: packages/commonwealth/client/scripts/views/pages/MyCommunityStake/MyCommunityStake.tsx (hunk 1 above)
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/pages/MyCommunityStake/MyCommunityStake.tsx]: done

**Review by lzach83 — APPROVED** [review]

**Review by Israellund — APPROVED** [review]
LGTM

**Pull request merged**